### PR TITLE
feat: bulk actions on alerts

### LINKED ISSUE #1495: [🔨 Enhancement]: bulk actions on every action from the alert menu (e.g. change status manually)

### PR DESCRIPTION
**Pull request by soylent-grin**
Closes #1495

## 📑 Description
Add bulk change status and self-assign for selected alerts. Not added "Run workflow" bulk action yet (now user is redirected to workflow logs after run, so this may be ambiguous)

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

## ℹ Additional Information

* Removed "Create or update preset" from bulk actions list; it requires space and is out of sync with main "create preset" functionality 
* Prevent crash in `Dockerfile.dev.ui` in case next binary already linked
* Adjust height of group actions (prevent table jump), more consistent action icons

### CODE REVIEW

File: docker/Dockerfile.dev.ui [hunk 1]
```diff
@@ -19,7 +19,7 @@ COPY ./keep-ui/ /app
 RUN npm install
 # Install next globally and create a symlink
 RUN npm install -g next
-RUN ln -s /usr/local/lib/node_modules/next/dist/bin/next /usr/local/bin/next
+RUN ln -s /usr/local/lib/node_modules/next/dist/bin/next /usr/local/bin/next || echo "next binary already linked to bin"
```
Comment (achivkun) [inline docker/Dockerfile.dev.ui:22]: I would put it to separate commit, as it don't seems to be related to this feature.

**Comment by talboren** [comment]
Closing for now as this seems stale. If this is still relevant, let's reopen it.
